File: implementations/Python/validate.py
#!/usr/bin/env python3

from LIVR import Validator
import sys, json

inputData = json.loads(sys.argv[1])
rules     = json.loads(sys.argv[2])

validator  = Validator.Validator(rules)

outputData = validator.validate(inputData)
errors     = validator.get_errors()

result = {'errors': errors} if errors else {'output': outputData}

print( json.dumps(result) )

File: implementations/Ruby/validate.rb
#!/usr/bin/env ruby

require "rubygems"
require "json"
require "LIVR"

LIVR.default_auto_trim("is_auto_trim")

input_data = JSON.parse(ARGV[0])
rules      = JSON.parse(ARGV[1])

validator  = LIVR.new(rules)

output_data = validator.validate(input_data)
errors      = validator.get_errors()

puts JSON.generate( output_data ? {:output => output_data} : {:errors => errors} )